java/src/main/java/be/tothepoint/Main.java

```diff
@@ -0,0 +1,42 @@
+package be.tothepoint;
```

Comment:
I'll rename it to be.playing.with.projections. If you want to do the presentation somewhere else then dddeurope, then you don't have to change it again :)